addon/helpers/d3-tick-format.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+import { timeFormat } from 'd3-time-format';
+
+export function d3TickFormat([tickFormat]) {
```

Comment:
I don't think we need a specific tick format helper. The d3 format helpers are useful in many places. It'd be better to have `format` and `format-time` helpers which also work as tick formats.